FastBank.Services/BankService/BankService.cs

```diff
@@ -1,22 +1,96 @@
 ﻿using FastBank.Domain;
 using FastBank.Domain.RepositoryInterfaces;
-using FastBank.Infrastructure.DTOs;
-using FastBank.Infrastructure.Context;
 using FastBank.Infrastructure.Repository;
+using FastBank.Services.MessageService;
+using FastBank.Services.TransactionService;
 
 namespace FastBank.Services.BankService
 {
     public class BankService : IBankService
     {
-        private readonly IRepository _repo;
+        private readonly IBankRepository _repoBank;
+        private readonly ITransactionService _transactionService;
+        private readonly IMessageService _messageService;
         public BankService()
         {
-            _repo = new Repository(new FastBankDbContext());
+            _repoBank = new BankRepository();
+            _transactionService = new TransactionService.TransactionService();
+            _messageService = new MessageService.MessageService();
+        }
+        public void CapitalReplenishment(User user)
+        {
+            //Here we should
+            //1.Validate user rights
+            //2.Show current bank capital amount
+            //3.Ask about amount to be replenished
+            //4 Confirm the operation
+            //5.Create transaction about action
+
+            if (user.Role != Roles.Banker)
+            {
+                Console.WriteLine("You have not access to this operation. Press any key for exit!");
+                Console.ReadKey();
+                return;
+            }
+
+            var bank = Get();
+
+            if (bank == null)
+            {
+                Console.WriteLine("Bank is not ready. Press any key for exit!");
```

Comment:
Please change the language of this error. 
"Bank is not ready" - doesn't mean anything. Change it for example to "Bank doesn't exist or bank isn't available"